src/current/_includes/releases/v24.1/v24.1.0-rc.1.md

```diff
@@ -0,0 +1,57 @@
+## v24.1.0-rc.1
+
+Release Date: May 8, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-0-rc-1-sql-language-changes">SQL language changes</h3>
+
+- Added a new session setting `optimizer_use_improved_multi_column_selectivity_estimate`, which if enabled, causes the [optimizer]({% link v24.1/cost-based-optimizer.md %}) to use an improved selectivity estimate for multi-column predicates. This setting will default to `true` on v24.2 and later, and `false` on earlier versions. [#123106][#123106]
```

Comment:
Is there a good link for "session settings"? Is that different than a session variable? Is there a good link for multi-column predicates?